rawspec.c

```diff
@@ -499,16 +508,41 @@ char tmp[16];
         fprintf(stderr, "TBIN     = %g\n",  raw_hdr.tbin);
 #endif // VERBOSE
 
+
+        // If processing a specific antenna
+        if(ant != -1) {
+          // Validate ant
+          if(ant > raw_hdr.nants) {
```

Comment:
Compare with `nants-1` (need to catch ants == nants as invalid)

Reply:
Hmm, this and the other request are the vestiges of confusion over whether to 0-index or 1-index the antennas. I'm happy to go with 0-indexing. Will add to the help of the CLI argument help.

Reply:
Line 513 should probably also check that `ant > -1` rather than `ant != -1`.  Need to treat any user supplied negative value for `ant` as being an explicit request for all antennas (i.e. in case the use something strange like `-a -2`).

Reply:
I caught and warned about it in line 515.

Reply:
> Hmm, this and the other request are the vestiges of confusion over whether to 0-index or 1-index the antennas. I'm happy to go with 0-indexing, as per the CLI argument's help string 🤦

Channels are 0-index so might as well be consistent with antennas.  To be fair, these are really "inputs" (or input pairs for dual pol) even though we call them "antennas", so `-a 0` selects the first input (pair) regardless of which antenna was connected to that input.  But I think it's OK to gloss over that detail here.